roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | egrep "SELinux status:.*enabled"
+  register: selinux_set
+  when: (ansible_os_family == "RedHat")
```

Comment:
Would that break things for ubuntu ? Whole point of this PR : 
1: It should run on Ubuntu
2: It should execute semanage command only when selinux is not disabled.